packages/data-driver-postgres/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import type { AbstractQuery, DataDriver } from '@directus/data/types';
+import { Readable } from 'node:stream';
+import { Pool } from 'pg';
+
+export interface DataDriverPostgresConfig {
+	connectionString: string;
+}
+
+export default class DataDriverPostgres implements DataDriver {
+	#config: DataDriverPostgresConfig;
+	#pool: Pool;
+
+	constructor(config: DataDriverPostgresConfig) {
+		this.#config = config;
+
+		this.#pool = new Pool({
+			connectionString: this.#config.connectionString,
+		});
+	}
+
+	async connect() {
+		await this.#pool.connect();
```

Comment:
According to the [node-postgres docs](https://node-postgres.com/apis/pool#poolconnect), `Pool.connect` can be used to acquire a client from the pool. Since this client is newer released, this will occupy one client at all times. So, I think we should remove this call.